baseframe/__init__.py

```diff
@@ -499,11 +500,11 @@ def get_timezone():
 def localized_country_list():
     """
     Returns a list of country codes (ISO3166-1 alpha-2) and country names,
-    localized to the user's locale as determined by :func:`get_preferred_locale`.
+    localized to the user's locale as determined by :func:`get_user_locale`.
 
     The localized list is cached for 24 hours.
     """
-    return _localized_country_list_inner(get_preferred_locale())
+    return _localized_country_list_inner(get_user_locale())
```

Comment:
This should use `get_locale()` as well.

Reply:
The inner function needs some smartness. It checks for `en` but will be fooled by `en_IN`. It needs to (a) expect a locale object and (b) look for locale match instead of string match.